scripts/require-sync/main.go

```diff
@@ -191,3 +199,34 @@ func opaClient(referential bool) (*opa.Client, error) {
 
 	return client, nil
 }
+
+func validateRequiresSyncDataContent(annotation string) (bool, error) {
```

Comment:
can we add a test please

Reply:
Sure - I added a simple test in `main_test.go`